catalog/build_ci/B001.md

```diff
@@ -2,7 +2,7 @@
 
 |**Code**           | **B001** |
 |**Title**          | All CI/CD scripts are in VC (CI Config in Code).|
-|**Description**    | |
+|**Description**    | No script belongs to CI/CD is out of version control.|
```

Comment:
I prefer: **No CI/CD script is out of version control**  
However, does this description add extra info at all?